src/main/groovy/wooga/gradle/paket/unity/tasks/PaketUnityInstall.groovy

```diff
@@ -17,26 +17,19 @@
 
 package wooga.gradle.paket.unity.tasks
 
-import groovy.transform.Internal
 import org.apache.commons.io.FileUtils
 import org.gradle.api.Action
 import org.gradle.api.file.FileCollection
 import org.gradle.api.file.FileVisitDetails
 import org.gradle.api.file.FileVisitor
 import org.gradle.api.internal.ConventionTask
-import org.gradle.api.tasks.Input
-import org.gradle.api.tasks.InputDirectory
-import org.gradle.api.tasks.InputFile
-import org.gradle.api.tasks.InputFiles
-import org.gradle.api.tasks.OutputDirectory
-import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.*
```

Comment:
This style of import is in general not how we do it.

Reply:
sorry my IDE suggested it. Does atlas have some templates/rules for IntelliJ for this ?

Reply:
No. Normally this never happened as the default (so I prosumed) was to not use wildcard imports. But a good point anyways since we should define a shared ruleset.